__tests__/Intergration.test.ts

```diff
@@ -83,7 +83,7 @@ describe("Integration tests", () => {
 
     it("should get all claims if no query params provided", async () => {
       // const response = await request(app).get("/api/claim"); // not working
-      const response = await request(app).get("/api/claims-all");
+      const response = await request(app).get("/api/claimsfeed2");
```

Comment:
are we having Candid and LInkedTrust app both use this same backend?  I think we should make the Candid routes separate if we find we have to make any changes, or layer something on them, or pass down the search - make sure both LinkedTrust and Candid can make changes freely, not affect each other, just in general thought